suite-native/atoms/src/Text.tsx

```diff
@@ -28,11 +28,14 @@ export const Text = ({
     color = 'gray1000',
     align = 'left',
     style,
+    children,
     ...otherProps
 }: TextProps) => {
     const { applyStyle } = useNativeStyles();
 
     return (
-        <RNText style={[applyStyle(textStyle, { variant, color, align }), style]} {...otherProps} />
+        <RNText style={[applyStyle(textStyle, { variant, color, align }), style]} {...otherProps}>
+            {children || ' '}
```

Comment:
why this?

Reply:
I did it to make the Text component keep always the same height even if no `children` was provided. Empty Text component is used for example in the new `FiatAmountFormatter`:
```tsx
 if (!value || isTestnetValue) return <Text />;
 ```

Reply:
I would rather not do that here, because it could lead to unexpected behavior. Do this only in places of usage to keep this atoms as much pure as possible. So rather do `return <Text>{' '}</Text>`

Reply:
fixed in c69893292